src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js

```diff
@@ -191,13 +194,31 @@ export const AzureComputeModalBase = ({
     )
   }
 
+  const sendCloudEnvironmentMetrics = () => {
+    const metricsEvent = Utils.cond(
+      [(viewMode === 'deleteEnvironment'), () => 'cloudEnvironmentDelete'],
+      //[(!!existingRuntime), () => 'cloudEnvironmentUpdate'], TODO: When update is available, include in metrics
+      () => 'cloudEnvironmentCreate'
+    )
+
+    //TODO: When update is available include existingRuntime in metrics.
+    Ajax().Metrics.captureEvent(Events[metricsEvent], {
+      ...extractWorkspaceDetails(workspace),
+      ..._.mapKeys(key => `desiredRuntime_${key}`, computeConfig),
+      desiredRuntime_region: computeConfig.region,
+      desiredRuntime_machineType: computeConfig.machineType,
+      desiredPersistentDisk_size: computeConfig.diskSize,
+      desiredPersistentDisk_type: 'Standard', //TODO: Hard coded - we currently don't send disk type.
```

Comment:
Would it make more sense to omit it, or send a `NaN` instead?

Reply:
My thinking is this - we default users to the standard disk type so we can send this with confidence, until we allow choosing a disk type.
I'm not opposed to omit it until the user has actual choices, though.